.github/workflows/release.yml

```diff
@@ -32,10 +32,29 @@ jobs:
         with:
           name: windows-artifacts
           path: /tmp/artifacts
-      - name: Set DRY_RUN_OPTION env variable
-        if: github.event.inputs.dry-run
-        run: echo "DRY_RUN_OPTION=--dry-run" >> $GITHUB_ENV
       - name: Deploy
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: dotnet run --project tools-shared/FakeItEasy.Deploy -- $DRY_RUN_OPTION -r "FakeItEasy/FakeItEasy.Analyzers" -t "${{ github.ref_name }}" -a "/tmp/artifacts/output"
+        run: |
+          if [[ "${{ github.event.inputs.dry-run }}" == "true" ]]; then
+            DRY_RUN_OPTION=--dry-run
+          fi
+          dotnet run --project tools-shared/FakeItEasy.Deploy -- $DRY_RUN_OPTION -r "${{ github.repository }}" -t "${{ github.ref_name }}" -a "/tmp/artifacts/output"

```

Comment:
Not related to the goal of this PR, but since I found an easier way to take `dry-run` into account, I also applied it here

Reply:
Ah, yes. Very nice. I like `github.repository` in there as well!